### ISSUE TITLE
How does the model control the problem of overfitting?

### ISSUE THREAD

**Issue by Maggione** [guestwalk/kaggle-2014-criteo#5]
As I see from the codes, the parameters about the validation path is set by the test set path. In addition, I can't see the operations of cross-validation in the codes.

**Comment by guestwalk** [comment]
Please check Section 4.3 in the following paper:
https://www.csie.ntu.edu.tw/~cjlin/papers/ffm.pdf

**Comment by Maggione** [comment]
Thank you for reply !
I have one more question. Do we need to avoid overfitting of GBDT, and how to avoid information leakage?

**Comment by guestwalk** [comment]
We also used a validation set to decide the number of trees and the depth of tree. I guess you may also need to do so.
